slack-base/src/test/java/com/hubspot/slack/client/models/events/json/EventDeserializerTest.java

```diff
@@ -80,6 +81,25 @@ public void itCanDeserChannelUnarchivedEvent() throws IOException {
     assertThat(event.getType()).isEqualTo(SlackEventType.CHANNEL_UNARCHIVE);
   }
 
+  @Test
+  public void itCanDeserUserChangeEventUserDeleted() throws IOException {
+    SlackUserChangeEvent event = fetchAndDeserializeSlackEvent("user_change_user_deleted_message.json").toDetailedEvent();
+    assertThat(event.getType()).isEqualTo(SlackEventType.USER_CHANGE);
+    assertThat(event.getUser().isDeleted().isPresent() && event.getUser().isDeleted().get());
+  }
+
+  @Test
+  public void itCanDeserUserChangeEventUsernameChanged() throws IOException {
+    SlackEvent event = fetchAndDeserializeSlackEvent("user_change_username_changed_message.json");
+    assertThat(event.getType()).isEqualTo(SlackEventType.USER_CHANGE);
+  }
+
+  @Test
+  public void itCanDeserUserChangeEventDisplayNameChanged() throws IOException {
+    SlackEvent event = fetchAndDeserializeSlackEvent("user_change_display_name_changed_message.json");
+    assertThat(event.getType()).isEqualTo(SlackEventType.USER_CHANGE);
+  }
```

Comment:
I might be missing something here, but there doesn't seem to be any way to know what, in particular, was changed for the user when the `user_changed` event comes through. So these json files with varying field values don't seem like they're really testing anything.

Reply:
That's exactly what it's testing, yeah. Failing to deserialize the events was what made my initial channels webhooks fail, so this just proves that these events can be properly deserialized

Reply:
So each of these tests have different names and different json files, but as far as the parsing goes, the objects are all the same. The only real (and testable) difference in the files looks to be the `type` field.  Seems like we don't need to test 3 varieties of the user json for the same type?